doc/elements.md

```diff
@@ -169,6 +169,39 @@ A = np.array([[1.23], [4.56]])
 If a variable `v` is a complex object, you should use `import prairielearn as pl` and `data['params'][params_name] = pl.to_json(v)`.
 
 
+## `pl-latex-matrix-display` element
+
+```html
+<pl-latex-matrix-display>
+    <variable params-name="A"  digits="2"> A = </variable>
+    <variable params-name="B"  digits="0"> </variable>
+</pl-latex-matrix-display>
```

Comment:
Why do we have this system with the element `pl-latex-matrix-display` containing multiple `variable` elements? It seems to me we could just have:
```
$A = <pl-latex-matrix params-name="A" digits="2"> <pl-latex-matrix params-name="B" digits="0">$
```
or something like this?

I know that `<pl-matrix-output>` uses a syntax like this, but I think that it's needed in that case because all the variables are going to end up displayed in a custom frame. For `<pl-latex-matrix-display>` we aren't doing this, so it just seems like extra complexity?

I'd be interested to hear @tbretl's thoughts about this.

Reply:
Cool, thanks!